config/repository_resource.go

```diff
@@ -279,23 +279,31 @@ func (r *repositoryResource) validate(ctx context.Context) (*ManagedResources, e
 	return &ManagedResources{Repositories: []ManagedRepository{r.managedRepository}}, nil
 }
 
-func contentsMatch(path, chksum string) (bool, error) {
+func contentsMatch(ctx context.Context, path, chksum string) (bool, error) {
 	file, err := os.OpenFile(path, os.O_RDONLY, 0644)
 	if err != nil {
 		if os.IsNotExist(err) {
+			clog.Debugf(ctx, "contentsMatch - File not found: %s", path)
```

Comment:
The logging framework already logs function/line with each message, do you need to repeat the function name?

Example: 
```sourceLocation: {
file: "config_task.go"
function: "github.com/GoogleCloudPlatform/osconfig/agentendpoint.(*configTask).run"
line: "451"
}```

Reply:
Thanks for letting me know, Done!